### ISSUE TITLE
problem with NEST 2.16.0 installation (make)

### ISSUE THREAD

**Issue by zhangysjlu** [nest/nest-simulator#1091]
after configure nest and compile nest by make, the error appears:
![screenshot from 2018-12-12 13-03-38](https://user-images.githubusercontent.com/13186004/49855351-f147e300-fe27-11e8-81f7-e06466bf655a.png)

**Comment by zhangysjlu** [comment]
Dear Alex,

Yes, I am using anaconda  and virtual environment. I have followed your suggestion and recompiled the nest. Now, it works. Thank you very much!

**Comment by pedroernesto** [comment]
The comments in this issue have been really useful to build NEST on miniconda2 as well. After running `make installcheck`, the installation gets these stats:
>  Total number of tests: 844
     Passed: 834
     Skipped: 10 (9 PyNEST)
     Failed: 0 (0 PyNEST) 

However, after `import nest`, submodule _sli_ can not be imported yet:

>  from nest import sli_run
---------------------------------------------------------------------------
> ImportError                               Traceback (most recent call last)
<ipython-input-2-07c3a5e0982c> in <module>()
----> 1 from nest import sli_run

> ImportError: cannot import name sli_run

**Comment by heplesser** [comment]
@pedroernesto I assume you have built NEST from the master branch in the github repo, not from the NEST 2.16.0 release. With #1058 we reorganized the low-level parts of the NEST Python interface. You now need to use
```
from nest.ll_api import sli_run
```

**Comment by pedroernesto** [comment]
yes, I built NEST from the master branch on GitHub. Thanks for the quick response.
